### PR TITLE
[READY] Lavaland Elite Bugfix and Tweaks

### PR DESCRIPTION
**Pull request by IndieanaJones**
## About The Pull Request

All this does is use the tumor's current name for its interactions, fix the issue of the broodmother's babies not spawning, and buffing her tentacles from 20-25 damage to 30-35 damage.  It also gives an alert message to freed elite about being friendly with the crew, and allows elites to re-enter the tumor to despawn themselves if they win.
## Why It's Good For The Game

Just some tweaks, a bugfix, and a small buff to an underperforming elite.  I was expecting more issues to crop up week one, but rather pleasantly that hasn't happened.  Re-entering the tumor is good if the elite has to leave so someone else can get the role.

## Changelog
:cl:
add: Elites who win their freedom can re-enter their tumor to despawn themselves, letting the tumor be re-activated to spawn a different elite.
tweak: A renamed pulsing tumor uses its new name in messages now.
tweak: Pulsing tumors now use their popped sprite after spawning an elite.
tweak: Boosted elites are now warned against being friendly with crew.
balance: Broodmother's tentacles do 30-35 damage from 20-25.
fix: Fixed the Broodmother being unable to create children.
/:cl:

### CODE REVIEW

**Comment by IndieanaJones** [comment]
I almost forgot the spriter reminded me the tumors have a popped sprite for after they spawn an elite, so we're going to use it now.

**Comment by IndieanaJones** [comment]
Oh, and I almost forgot about this too: I added a note when an elite wins their freedom that they shouldn't be friendly with crew.  I was seeing some cases of happening pretty often, so the note is there to discourage such from happening further.

**Comment by kriskog** [comment]
Not everyone might wish to take the roll if it means spending the rest of the round as an elite and feeling shitty about suiciding or ghosting. Add a way to crawl back into your tumor when you finished murdering the miner, so others can activate it and get a new roll later?

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm [hunk 1]
```diff
@@ -323,7 +325,7 @@ obj/structure/elite_tumor/proc/onEliteWon()
 	smooth = SMOOTH_TRUE
 	layer = BELOW_MOB_LAYER
 	var/mob/living/carbon/human/activator = null
-	var/mob/living/simple_animal/hostile/asteroid/elite/ourelite = null
+	var/mob/living/ourelite = null
```
Comment (AnturK) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm]: What's this about

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm [hunk 2]
```diff
@@ -281,6 +282,7 @@ obj/structure/elite_tumor/proc/onEliteWon()
 		SEND_SOUND(mychild, sound('sound/effects/magic.ogg'))
 		to_chat(mychild, "<span class='boldwarning'>As the life in the activator's eyes fade, the forcefield around you dies out and you feel your power subside.\nDespite this inferno being your home, you feel as if you aren't welcome here anymore.\nWithout any guidance, your purpose is now for you to decide.</span>")
 		to_chat(mychild, "<b>Your max health has been halved, but can now heal by standing on your tumor.  Note, it's your only way to heal.\nBear in mind, if anyone interacts with your tumor, you'll be resummoned here to carry out another fight.  In such a case, you will regain your full max health.\nAlso, be weary of your fellow inhabitants, they likely won't be happy to see you!</b>")
+		to_chat(mychild, "<span class='big bold'>Note that you are a lavaland monster, and thus not allied to the station.  You should not cooperate or act friendly with any station crew unless under extreme circumstances!</span>")
```
Comment (ATH1909) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm]: might wanna add a clarification for rezzed elites that they are/have become stationed-aligned if they're rezzed by someone station-aligned (i.e. a non-traitor miner)

**Comment by IndieanaJones** [comment]
> Not everyone might wish to take the roll if it means spending the rest of the round as an elite and feeling shitty about suiciding or ghosting. Add a way to crawl back into your tumor when you finished murdering the miner, so others can activate it and get a new roll later?

Perfectly feasible, I can do that.


> I wasn’t able to butcher the mini-colossus. Admin had to gib it so I could get the cloak. Check it out please.

You're not supposed to butcher him for his item, it spawns in the chest from the tumor 50% of the time.  If the elite came into existence from admemes or wasn't challenged at the tumor when it died, it won't drop anything.  Non-boosted elites also don't have a chance to drop their unique loot item.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm (hunk 1 above)
Comment (IndieanaJones) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm]: Leftover accident from some testing, though keeping doesn't do anything.  I'll change it back.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm (hunk 2 above)
Comment (IndieanaJones) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm]: Can do.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm [hunk 3]
```diff
@@ -281,6 +293,7 @@ obj/structure/elite_tumor/proc/onEliteWon()
 		SEND_SOUND(mychild, sound('sound/effects/magic.ogg'))
```
Comment (actioninja) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm]: I didn't catch this in the original pr but use `mychild.playsound_local` instead

**Review by actioninja — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm (hunk 3 above)
Comment (IndieanaJones) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm]: Just here, or everywhere SEND_SOUND is used?

Comment (AnturK) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm]: Generally everywhere, send sound is just raw sound sending, playsound/_local is wrapper that should be used anywhere that handles deafness/distance etc.

**Pull request merged**